Source/WebKit/UIProcess/Cocoa/NavigationState.mm

```diff
@@ -535,7 +535,7 @@ static void tryInterceptNavigation(Ref<API::NavigationAction>&& navigationAction
             case _WKNavigationActionPolicyAllowWithoutTryingAppLink:
                 trySOAuthorization(WTFMove(navigationAction), webPageProxy, [localListener = WTFMove(localListener), websitePolicies = WTFMove(apiWebsitePolicies)] (bool optimizedLoad) {
```

Comment:
Nit: We should rename `optimizedLoad` to `interceptedNavigation` to match other methods. `optimizedLoad` was a temporary name we were using before the feature was finalized.